views/postgres/show.erb

```diff
@@ -108,6 +110,71 @@ delete_perm = has_permission?("Postgres:delete", @pg[:id]) %>
       </div>
     <% end %>
   </div>
+  <% if @project.get_ff_postgres_metrics %>
+    <div>
+      <div class="md:flex md:items-center md:justify-between pb-2 lg:pb-4">
+        <div class="min-w-0 flex-1">
+          <h3 class="text-2xl font-bold leading-7 text-gray-900 sm:truncate sm:text-2xl sm:tracking-tight">
+            Metrics
+          </h3>
+        </div>
+      </div>
+      <div
+        id="metrics-container"
+        class="p-4 overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 bg-white divide-y divide-gray-200"
+      >
+        <% if @pg[:state] == "running" %>
+          <div class="mb-4 flex flex-wrap items-end gap-4 justify-end">
+            <div class="relative inline-block text-left">
+              <div class="relative inline-block text-left">
+                <select
+                  id="time-range"
+                  class="py-1.5 md:align-top rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50"
+                  name="time-range"
+                >
+                  <option value="15m">Last 15 minutes</option>
+                  <option value="1h" selected>Last 1 hour</option>
+                  <option value="3h">Last 3 hours</option>
+                  <option value="6h">Last 6 hours</option>
+                  <option value="12h">Last 12 hours</option>
+                  <option value="1d">Last 1 day</option>
+                  <option value="2d">Last 2 days</option>
+                  <option value="7d">Last 7 days</option>
+                  <option value="30d">Last 30 days</option>
+                </select>
+                <%== part("components/button", attributes: {id: "refresh-button"}, text: "  Refresh", type: "primary", icon: "hero-refresh") %>
+              </div>
+            </div>
+          </div>
+          <div class="grid grid-cols-1 xl:grid-cols-2 gap-4 !border-t-0">
+            <% Metrics::POSTGRES_METRICS.each do |key, metric| %>
+              <div class="relative p-4 bg-gray-50 rounded-lg border border-gray-100">
+                <h4 class="text-sm font-medium text-gray-900 mb-2"><%= metric.name %></h4>
+                <p class="text-xs text-gray-500 mt-2"><%= metric.description %></p>
+                <div
+                  class="metric-chart ml-4 h-64 w-full"
+                  id="<%= key %>-chart"
+                  data-metric-key="<%= key %>"
+                  data-metric-unit="<%= metric.unit %>"
+                >
+                </div>
+              </div>
+            <% end %>
+            <!-- Hidden div with color pallete so that echarts can use it -->
```

Comment:
Is this to ensure tailwind generates class definitions? If so, you can add them to `tailwind.config.js` as safelist.

Reply:
yes, let me add these there